python/benchmarks/common.py

```diff
@@ -37,7 +37,7 @@ def generate_random_floats_dataframe(num_rows, num_cols):
     return pd.DataFrame(data, columns=columns)
 
 
-def generate_benchmark_df(n, freq="T", end_timestamp="1/1/2023"):
+def generate_benchmark_df(n, freq="min", end_timestamp="1/1/2023"):
```

Comment:
This is the same, isn't it?

Reply:
Yes, but T is apparently deprecated, and we get warnings from pandas which are quite annoying.